dp/cloud/python/magma/configuration_controller/run.py

```diff
@@ -83,7 +83,7 @@ def run():
         max_overflow=config.SQLALCHEMY_ENGINE_MAX_OVERFLOW,
     )
     session_manager = SessionManager(db_engine=db_engine)
-    ssl_validator = CRLValidator(urls=[config.SAS_URL]) if config.VERIFY_SAS_CRL else None
+    ssl_validator = CRLValidator(urls=[config.SAS_URL])
```

Comment:
Can crl verification still be disabled?

Reply:
It's required by certification process to perform CRL validation when making requests to SAS.   
It's always enabled and cannot be disabled, but it fetches only SAS cert and CRLs and is used only in `RequestRouter.post_to_sas` method.